<ui version="4.0" >
 <class>chatWindowClass</class>
 <widget class="QWidget" name="chatWindowClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>chatWindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <property name="margin" >
    <number>3</number>
   </property>
   <item row="0" column="0" colspan="7" >
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QLabel" name="avatarLabel" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Note:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3" >
       <property name="spacing" >
        <number>3</number>
       </property>
       <item>
        <widget class="QLabel" name="nameLabel" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="textInteractionFlags" >
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="noteEdit" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <property name="spacing" >
        <number>3</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <widget class="QLabel" name="uinLabel" >
           <property name="minimumSize" >
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="frameShape" >
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="alignment" >
            <set>Qt::AlignCenter</set>
           </property>
           <property name="textInteractionFlags" >
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="infoButton" >
           <property name="maximumSize" >
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset resource="../../../qutim.qrc" >
             <normaloff>:/icons/crystal_project/contactinfo.png</normaloff>:/icons/crystal_project/contactinfo.png</iconset>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="clientLabel" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>20</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>140</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>140</width>
           <height>20</height>
          </size>
         </property>
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="7" >
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="handleWidth" >
      <number>5</number>
     </property>
     <property name="childrenCollapsible" >
      <bool>false</bool>
     </property>
     <widget class="icqTextBrowser" name="messageChat" >
      <property name="html" >
       <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Verdana'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>&lt;/body>&lt;/html></string>
      </property>
      <property name="openExternalLinks" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QToolButton" name="historyButton" >
           <property name="maximumSize" >
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>History</string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset resource="../../../qutim.qrc" >
             <normaloff>:/icons/crystal_project/history.png</normaloff>:/icons/crystal_project/history.png</iconset>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="emoticonButton" >
           <property name="maximumSize" >
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset resource="../../../qutim.qrc" >
             <normaloff>:/icons/crystal_project/emoticon.png</normaloff>:/icons/crystal_project/emoticon.png</iconset>
           </property>
           <property name="shortcut" >
            <string>Ctrl+M</string>
           </property>
           <property name="popupMode" >
            <enum>QToolButton::InstantPopup</enum>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="fontButton" >
           <property name="maximumSize" >
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset resource="../../../qutim.qrc" >
             <normaloff>:/icons/crystal_project/customfonts.png</normaloff>:/icons/crystal_project/customfonts.png</iconset>
           </property>
           <property name="popupMode" >
            <enum>QToolButton::InstantPopup</enum>
           </property>
           <property name="toolButtonStyle" >
            <enum>Qt::ToolButtonIconOnly</enum>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="backgroundButton" >
           <property name="maximumSize" >
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset resource="../../../qutim.qrc" >
             <normaloff>:/icons/crystal_project/backcolor.png</normaloff>:/icons/crystal_project/backcolor.png</iconset>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="imageButton" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="maximumSize" >
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">Send image (smaller then 7.6 KB)&lt;/p>&lt;/body>&lt;/html></string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset resource="../../../qutim.qrc" >
             <normaloff>:/icons/crystal_project/image.png</normaloff>:/icons/crystal_project/image.png</iconset>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="sendFileButton" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="maximumSize" >
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>Send file</string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset resource="../../../qutim.qrc" >
             <normaloff>:/icons/crystal_project/save_all.png</normaloff>:/icons/crystal_project/save_all.png</iconset>
           </property>
           <property name="shortcut" >
            <string>Ctrl+F</string>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="translateButton" >
           <property name="maximumSize" >
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset resource="../../../qutim.qrc" >
             <normaloff>:/icons/translate.png</normaloff>:/icons/translate.png</iconset>
           </property>
           <property name="shortcut" >
            <string>Ctrl+T</string>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="countLabel" >
           <property name="minimumSize" >
            <size>
             <width>48</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>16777215</width>
             <height>22</height>
            </size>
           </property>
           <property name="frameShape" >
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="text" >
            <string>0</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
           <property name="textInteractionFlags" >
            <set>Qt::LinksAccessibleByMouse</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="typingLabel" >
           <property name="minimumSize" >
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="frameShape" >
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="icqTextEdit" name="messageEdit" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet" >
          <string notr="true" />
         </property>
         <property name="acceptRichText" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="ownAvatarLabel" >
     <property name="minimumSize" >
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="3" >
    <widget class="QToolButton" name="quoteButton" >
     <property name="maximumSize" >
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="toolTip" >
      <string>Quote selected text</string>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset resource="../../../qutim.qrc" >
       <normaloff>:/icons/crystal_project/quote.png</normaloff>:/icons/crystal_project/quote.png</iconset>
     </property>
     <property name="shortcut" >
      <string>Ctrl+Q</string>
     </property>
     <property name="autoRaise" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="4" >
    <widget class="QToolButton" name="clearButton" >
     <property name="maximumSize" >
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="toolTip" >
      <string>Clear chat</string>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset resource="../../../qutim.qrc" >
       <normaloff>:/icons/crystal_project/edituser.png</normaloff>:/icons/crystal_project/edituser.png</iconset>
     </property>
     <property name="autoRaise" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="5" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>234</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="6" >
    <widget class="QPushButton" name="sendButton" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Send</string>
     </property>
     <property name="icon" >
      <iconset resource="../../../qutim.qrc" >
       <normaloff>:/icons/crystal_project/message.png</normaloff>:/icons/crystal_project/message.png</iconset>
     </property>
     <property name="autoDefault" >
      <bool>false</bool>
     </property>
     <property name="default" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QToolButton" name="sendTypingButton" >
     <property name="maximumSize" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="toolTip" >
      <string>Send typing notifications</string>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset resource="../../../qutim.qrc" >
       <normaloff>:/icons/crystal_project/typing.png</normaloff>:/icons/crystal_project/typing.png</iconset>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="autoRaise" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QToolButton" name="sendOnEnterButton" >
     <property name="maximumSize" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="toolTip" >
      <string>Send on enter</string>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset resource="../../../qutim.qrc" >
       <normaloff>:/icons/crystal_project/key_enter.png</normaloff>:/icons/crystal_project/key_enter.png</iconset>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="autoRaise" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>icqTextEdit</class>
   <extends>QTextEdit</extends>
   <header>protocol/oscar/icq/icqtextedit.h</header>
  </customwidget>
  <customwidget>
   <class>icqTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header>protocol/oscar/icq/icqtextbrowser.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>messageChat</tabstop>
 </tabstops>
 <resources>
  <include location="../../../qutim.qrc" />
 </resources>
 <connections/>
</ui>
